File: examples/control_flow.rs
fn main() {
	const say_hello: bool = false;

	if say_hello {
		println!("Hello World!");
	} else {
		println!("Bye World!");
	}

	if !say_hello {
		println!("`say_hello` is not `true`");
	}

	if true {
		//
	} else if false {
		println!("testing")
	} else {
		println!("nope")
	}

	let mut x = 0;
	loop {
		println!("Round: {x}");
		x += 1;
		if x > 99 {
			break;
		}

		let mut y = 0;
		loop {
			print!(y);
			y = y + 1;
			if y >= 99 {
				println!();
				break;
			}
		}
	}
}

File: examples/control_flow.py
say_hello = False

if say_hello:
	print("Hello World!")
else:
	print("Bye World!")

if not say_hello:
	print("`say_hello` is not `true`")

if True:
	""
elif False:
	print("testing")
else:
	print("nope")

x = 0
while True:
	print("Round:", x)
	x += 1
	if x > 99:
		break

	y = 0
	while True:
		print(y)
		y += 1
		if y >= 99:
			break
